components/zoom/actions/add-meeting-registrant/add-meeting-registrant.js

```diff
@@ -0,0 +1,244 @@
+const { axios } = require("@pipedreamhq/platform");
```

Comment:
We need to document this, but you should actually use `@pipedream/platform` (no `hq`). We only acquired the `pipedream` namespace recently, so we're publishing the newest version there.